compatibility/sandbox-migration/util.bzl

```diff
@@ -7,6 +7,9 @@ load("//bazel_tools:versions.bzl", "versions")
 def runfiles(ver):
     return ["@daml-sdk-{}//:daml".format(ver)] + (["@daml-sdk-{}//:sandbox-on-x".format(ver)] if versions.is_at_least("2.0.0", ver) else [])
 
+def oracle_versions(vers):
+    return [v for v in vers if versions.is_at_least("1.18.0", v)]
```

Comment:
oh, does that mean we need to provide backwards compatibility for persistence for Oracle form version 1.18 already?
then we cannot squash he Oracle scripts anymore neither :thinking:

Reply:
correct, we need to be compatible from 1.18